openmetadata-ui/src/main/resources/ui/src/components/PermissionProvider/PermissionProvider.tsx

```diff
@@ -74,13 +80,22 @@ const PermissionProvider: FC<PermissionProviderProps> = ({ children }) => {
     return AppState.getCurrentUserDetails();
   }, [AppState.userDetails, AppState.nonSecureUserDetails]);
 
+  const handledVerifiedUser = () => {
```

Comment:
Here method name is `handledVerifiedUser` but I can not see any user-related stuff. Can you give an appropriate name to the method?

Reply:
Updating!